scripts/openmc-ace-to-hdf5

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import xml.etree.ElementTree as ET
+import warnings
+
+import openmc.data
+
+description = """
+This script can be used to create HDF5 nuclear data libraries used by
+OpenMC. There are four different ways you can specify ACE libraries that are to
+be converted:
+
+1. List each ACE library as a positional argument. This is very useful in
+   conjunction with the usual shell utilities (ls, find, etc.).
+2. Use the --xml option to specify a pre-v0.9 cross_sections.xml file.
+3. Use the --xsdir option to specify a MCNP xsdir file.
+4. Use the --xsdata option to specify a Serpent xsdata file.
+
+The script does not use any extra information from cross_sections.xml/ xsdir/
+xsdata files to determine whether the nuclide is metastable. Instead, the
+--metastable argument can be used to specify whether the ZAID naming convention
+follows the NNDC data convention (1000*Z + A + 300 + 100*m), or the MCNP data
+convention (essentially the same as NNDC, except that the first metastable state
+of Am242 is 95242 and the ground state is 95642).
+
+"""
+
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
```

Comment:
I can see how the authors put the description right above the class definition so that it serves a dual purpose -- a class comment as well as an ArgParse description. If the code authors do not intend to follow PEP8 standards, this is ok, but a PEP8 linter would complain about not having two spaces before the class definition. I would recommend adding two spaces before the class definition, and one space before the `description` variable is declared for cleanliness and clear separation between imports, global variable(s), and the class def.